letsApply/Controllers/Authentication/SignUpWithEmail/SignUpViewController.swift

```diff
@@ -6,55 +6,18 @@
 //
 
 import Foundation
-import FirebaseAuth
 import UIKit
+import FirebaseAuth
```

Comment:
Don't forget that if you are importing the in a ViewController something is wrong. It should be in the SignUpViewModel